internal/util/grpcclient/client.go

```diff
@@ -328,12 +341,16 @@ func (c *ClientBase[T]) call(ctx context.Context, caller func(client T) (any, er
 		}
 		ret, callErr = caller(client)
 		if callErr != nil {
-			if funcutil.IsGrpcErr(callErr) ||
+			if funcutil.IsGrpcErr(callErr, codes.Unavailable, codes.Aborted, codes.FailedPrecondition) ||
```

Comment:
I don't like the idea to list all the retriable errors.
But you can list errors you don't want to reset connection